ios/Handlers/RNFlingHandler.m

```diff
@@ -27,5 +27,11 @@ - (void)configure:(NSDictionary *)config
     }
 }
 
+- (RNGestureHandlerEventExtraData *)eventExtraData:(UISwipeGestureRecognizer *)recognizer
```

Comment:
Note that this changes behavior for Fling (same for long press). They've previously been using base class implementation from `RNGestureHandler.m`:
![image](https://user-images.githubusercontent.com/726445/39289160-3387740e-492b-11e8-9fb8-75178de7f117.png)

It sends position and absolute position. You perhaps want to add "withNumberOfTouches" there instead